src/envoy/http/authn/filter_context.h

```diff
@@ -56,8 +57,10 @@ class FilterContext : public Logger::Loggable<Logger::Id::filter> {
   // Returns the authentication result.
   const istio::authn::Result& authenticationResult() { return result_; }
 
-  // Accessor to headers.
-  HeaderMap* headers() { return headers_; }
+  // Acessor to dynamic metadata.
+  const envoy::api::v2::core::Metadata& dynamic_metadata() const {
```

Comment:
consistent name style:  dynamicMetadata?

Reply:
Remove this function actually, as all needs is satisfied by getJwtPayload function below.